recipes/bob.core/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "2.1.2" %}
+
+package:
+  name: bob.core
+  version: {{ version }}
+
+source:
+  fn: bob.core-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.core/bob.core-{{ version }}.zip
+  md5: 26c12e1384944ff3873b08409609d151
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
+  script: LINKFLAGS="$LINKFLAGS -stdlib=libc++ -mmacosx-version-min=10.7 " CXXFLAGS="$CXXFLAGS -stdlib=libc++ -mmacosx-version-min=10.7 " python -B setup.py install --single-version-externally-managed --record record.txt  # [osx]
```

Comment:
I'm sorry, but it is hard to read and you are asking me to help you figure out what is wrong. While I'm happy to help, this is just a bit painful to read this long one line string. Please break it out before we continue.